### PR TITLE
Add permissions and UHC docs

### PR DESCRIPTION
**Pull request by eljhkrr**
Resolves https://github.com/medic/cht-docs/issues/400
Resolves https://github.com/medic/cht-docs/issues/82

### CODE REVIEW

File: content/en/apps/reference/app-settings/user-permissions.md [hunk 1]
```diff
@@ -10,8 +10,55 @@ relevantLinks: >
 keywords: user-roles user-permissions
 ---
 
+Permissions are defined by the `permissions` object in the `base_settings.json` file. Permissions can also be configured using the App Management app.
+Each permission is defined in terms of an array of user role identifiers that have the permission granted.
```
Comment (smbuthia) [inline content/en/apps/reference/app-settings/user-permissions.md:14]:
```suggestion
Each permission is defined as an array of user role identifiers that have the permission granted.
```

**Review by smbuthia — CHANGES_REQUESTED** [review]
Thanks for adding these. I've added some comments/suggestions inline.

File: content/en/apps/guides/forms/uhc-mode.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  apps/reference/app-settings/user-roles
+  apps/reference/app-settings/user-permissions
+---
+
+_Introduced in v2.18.0_
+
+UHC mode empowers CHWs to provide equitable and timely care to families in their catchment area. The Community Health Toolkit supports this use-case by displaying the number of
+visits made to a household and highlighting households which haven't met their visit goal in red at the top of the contact list.
+
+![UHC mode](UHC.gif)
```
[the remark this answers is not in the document]
Comment (eljhkrr) [reply content/en/apps/guides/forms/uhc-mode.md]: Updated

File: content/en/apps/guides/forms/uhc-mode.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+The date last visited is colored red whenever the date is 30 days or more in the past. The date last visited is displayed as a relative date. Visits that happened today will say "Visited today", visits that happened yesterday will say "Visited yesterday". This period is described in terms of "days" up until two months, so "Visited 3 days ago" or "visited 36 days ago". After two months, we simply say "2 months" or "3 months".
+
+The date of the last visit is displayed and contacts can be sorted by last visit date to allow the CHW plan their work.
+
+![Contact sort](person-contact-form-survey.png)
```
[the remark this answers is not in the document]
Comment (eljhkrr) [reply content/en/apps/guides/forms/uhc-mode.md]: Updated

File: content/en/apps/guides/forms/uhc-mode.md [hunk 4]
```diff
@@ -14,13 +14,13 @@ _Introduced in v2.18.0_
 UHC mode empowers CHWs to provide equitable and timely care to families in their catchment area. The Community Health Toolkit supports this use-case by displaying the number of
 visits made to a household and highlighting households which haven't met their visit goal in red at the top of the contact list.
 
-![UHC mode](UHC.gif)
+{{< figure src="UHC.gif" link="UHC.gif" alt="UHC mode screenshot" title="UHC mode in use" >}}
 
 The date last visited is colored red whenever the date is 30 days or more in the past. The date last visited is displayed as a relative date. Visits that happened today will say "Visited today", visits that happened yesterday will say "Visited yesterday". This period is described in terms of "days" up until two months, so "Visited 3 days ago" or "visited 36 days ago". After two months, we simply say "2 months" or "3 months".
```
Comment (smbuthia) [inline content/en/apps/guides/forms/uhc-mode.md]:
```suggestion 
{{< figure src="UHC.gif" link="UHC.gif" alt="UHC mode screenshot" class="right col-6 col-lg-3" >}}

UHC mode empowers CHWs to provide equitable and timely care to families in their catchment area. The Community Health Toolkit supports this use-case by displaying the number of
visits made to a household and highlighting households that haven't met their visit goal in red at the top of the contact list.

The date last visited is colored red whenever the date is 30 days or more in the past. The date last visited is displayed as a relative date. This period is described in terms of "days" up until two months, so "Visited 3 days ago" or "Visited 36 days ago". After two months, we simply say "2 months" or "3 months".
```

File: content/en/apps/guides/forms/uhc-mode.md [hunk 5]
```diff
@@ -14,13 +14,13 @@ _Introduced in v2.18.0_
...
 
 The date of the last visit is displayed and contacts can be sorted by last visit date to allow the CHW plan their work.
 
-![Contact sort](sort-dropdown.png)
+{{< figure src="sort-dropdown.png" link="sort-dropdown.png" alt="Contact sorting screenshot" title="Contact sorting in UHC mode" >}}
```
Comment (smbuthia) [inline content/en/apps/guides/forms/uhc-mode.md]:
```suggestion
### Sorting

{{< figure src="sort-dropdown.png" link="sort-dropdown.png" alt="Contact sorting screenshot" class="right col-6 col-lg-3" >}}

The date of the last visit is displayed and contacts can be sorted by the last visit date to allow the CHW to plan their work.
```

File: content/en/apps/guides/forms/uhc-mode.md (hunk 4 above)
Comment (eljhkrr) [reply content/en/apps/guides/forms/uhc-mode.md]: Updated

File: content/en/apps/guides/forms/uhc-mode.md (hunk 5 above)
Comment (eljhkrr) [reply content/en/apps/guides/forms/uhc-mode.md]: Updated

**Review by smbuthia — APPROVED** [review]
Looks good. Thanks.

**Pull request merged**